foundation/foundation-configuration-store/pom.xml

```diff
@@ -149,6 +149,12 @@
             <artifactId>kubernetes-client</artifactId>
             <version>6.10.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.flipkart.zjsonpatch</groupId>
+            <artifactId>zjsonpatch</artifactId>
+            <version>0.4.16</version>
```

Comment:
S: I'm very reluctant about adding a dependency on a pre 1.0 version of a library that hasn't been updated in nearly a year.

Reply:
yeah but it does seem this is already included in the quarkus-resteasy-jackson api so it actually can be cleaned up.

Reply:
No we should leave it if we're interfacing with it directly.  Best practice is to always list the dependencies you directly use as direct dependencies, even when they're available transitively.  This makes it much easier to maintain as transitive dependencies are removed/updated.